RocketApp/app/src/main/java/com/example/rocketapp/Experiment.java

```diff
@@ -15,6 +15,10 @@
     protected ArrayList<? extends Trial> trialsArrayList = new ArrayList<>();
     private ArrayList<Question> questionsArrayList = new ArrayList<>();
 
+    // TODO: Switch to using two boolean instead
```

Comment:
Active and Published mean the same thing though.
Active -> Anyone subscribed can add trials and view results, owner can end or unpublish
Ended -> Anyone can view results, owner can unpublish
Unpublished -> Only owner can see